apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationFooter.tsx

```diff
@@ -112,7 +128,12 @@ export const ApplicationDetailsFooter = ({
 
   const isMyApp = isMyApplication(entity);
   const isAdmin = useAppSelector(AuthSelectors.selectIsAdmin);
-  const isPublicApp = isEntityIdPublic(entity);
+
+  const isPublicEntity = isEntityIdPublic(entity);
```

Comment:
it would be nice to name it differently from `isPublicApp`
```suggestion
  const hasPublicId = isEntityIdPublic(entity);
```